cpp/src/aztec/honk/sumcheck/polynomials/multivariates.test.cpp

```diff
@@ -4,51 +4,12 @@
 #include <gtest/gtest.h>
 #include <numeric/random/engine.hpp>
 
+#pragma GCC diagnostic ignored "-Wunused-variable"
+
 using namespace honk::sumcheck;
 namespace test_sumcheck_polynomials {
 
-template <class FF> class MultivariatesTests : public testing::Test {
-  public:
-    // template <size_t num_polys, size_t multivariate_d>
```

Comment:
Very old test. Updating it probably as easy as just writing a new test from scratch.